app/templates/buyers/review_brief.html

```diff
@@ -0,0 +1,91 @@
+{% extends "_base_page.html" %}
+{% from "components/tabs/macro.njk" import govukTabs %}
+
+{% block pageTitle %}{{ brief.title }} [DRAFT] - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with
+    items = [
+      {
+          "link": url_for('external.index'),
+          "label": "Digital Marketplace"
+      },
+      {
+          "link": url_for('external.list_opportunities', framework_family=brief.framework.family),
+          "label": "Supplier opportunities"
+      },
+      breadcrumbs
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+
+{% block main_content %}
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <header class="page-heading-smaller">
+      <h1>Review your requirements</h1>
+    </header>
+  </div>
+</div>
+
+<div class="grid-row">
+  <div class="column-one-whole">
```

Comment:
Could we swap these out for govuk-frontend grids `govuk-grid-row` & `govuk-grid-column-full`?

Here is a really good reference to the classes we should be swapping to layout-and-grid-system-class-names

Reply:
Sure - the new classes seem to work ok